instance/migrations/0076_openedxappserver__is_active.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.3 on 2017-02-15 05:10
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def populate_is_active(apps, schema_editor):
+    '''
+    Mark all OpenEdXInstance.active_appserver with OpenEdXAppServer.is_active = True
+    '''
+
+    OpenEdXInstance = apps.get_model("instance", "OpenEdXInstance")
+    for instance in OpenEdXInstance.objects.exclude(active_appserver__isnull=True):
```

Comment:
There's so many gotchas with data migrations, I didn't think about the multiple database issue.  Thanks for pointing this out, @smarnach! 

But I'm inclined to leave it be. When we add multiple databases, we'll have to fix all our data migrations.